Bottom: 34a422e20b6864421c4dcbf249875027c9956b02
Top:    47f8fbb306d277301161e94470a45895de0fbb36
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-26 23:31:37 -0700

Refresh of mpi-finalize

---

diff --git a/src/rsocket.c b/src/rsocket.c
index c53d5d5..3ba0a74 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1723,7 +1723,6 @@ int rshutdown(int socket, int how)
 	int ctrl, ret = 0;
 
 	rs = idm_at(&idm, socket);
-	printf("rshutdown %d - %p {\n", socket, rs);
 	if (how == SHUT_RD) {
 		rs_shutdown_state(rs, rs_connect_rd);
 		return 0;
@@ -1753,21 +1752,14 @@ int rshutdown(int socket, int how)
 		ret = rs_post_write(rs, 0, NULL, 0,
 				    rs_msg_set(RS_OP_CTRL, ctrl),
 				    0, 0, 0);
-		if (ret)
-			printf("rs_post_write failed %d %s\n", ret, strerror(errno));
 	}
 
 	if (!rs_all_sends_done(rs) && !(rs->state & rs_error))
-	{
-		printf("  rshutdown - need sends %d - %p\n", socket, rs);
 		rs_process_cq(rs, 0, rs_all_sends_done);
-		printf("  rshutdown - have sends %d - %p\n", socket, rs);
-	}
 
 	if ((rs->fd_flags & O_NONBLOCK) && (rs->state & rs_connected))
 		rs_set_nonblocking(rs, 1);
 
-	printf("} rshutdown %d - %p\n", socket, rs);
 	return 0;
 }
 
@@ -1776,11 +1768,9 @@ int rclose(int socket)
 	struct rsocket *rs;
 
 	rs = idm_at(&idm, socket);
-	printf("rclose %d - %p {\n", socket, rs);
 	if (rs->state & rs_connected)
 		rshutdown(socket, SHUT_RDWR);
 
-	printf("} rclose %d - %p\n", socket, rs);
 	rs_free(rs);
 	return 0;
 }
